style.css

```diff
@@ -1046,512 +961,599 @@ border: 1px solid #fff;
+    align-items: center;
+    padding: 0;
+    gap: 12px;
+    width: 297px;
+    height: 40px;
+  }
 
-.tonic .year {
-  width: 38px;
-  height: 24px;
-  font-weight: 400;
-  font-size: 18px;
-  line-height: 24px;
-}
+  .tonic .canopy {
+    width: 75px;
+    height: 24px;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
+  }
 
-.paragraph2 {
-  width: 496px;
-  height: 50px;
-  font-weight: 400;
-  font-size: 16px;
-  line-height: 24px;
-}
+  .tonic .dot {
+    width: 8px;
+    height: 8px;
+  }
 
-.resources {
-  gap: 8px;
-  width: 203px;
-  height: 24px;  
-}
+  .tonic .back {
+    width: 120px;
+    height: 24px;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+  }
 
-.action {
-  width: 496px;
-  height: 153px;  
-}
+  .tonic .year {
+    width: 38px;
+    height: 24px;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+  }
 
-.normalbutton {
-  width: 124px;
-  height: 48px;
-  padding: 12px;
-  gap: 10px;
-}
+  .paragraph2 {
+    width: 496px;
+    height: 50px;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+  }
 
-.seeproject{
-  font-weight: 500;
-  font-size: 17px;
-  line-height: 24px;
-}
+  .resources {
+    gap: 8px;
+    width: 203px;
+    height: 24px;
+  }
 
-.card2 {
-  display: flex;
-  flex-direction: row;
-  width: 1156px;
-}
+  .action {
+    width: 496px;
+    height: 153px;
+  }
 
-.avator4 {
-  width: 544px;
-  height: 448px;
-  left: 590px;
-  top: 24px;
-  order: 2;
-}
+  .normalbutton {
+    width: 124px;
+    height: 48px;
+    padding: 12px;
+    gap: 10px;
+  }
 
-.container2 {
-  order: 1;
-  padding: 24px;
-  gap: 24px;
-  /* position: absolute; */
-  width: 544px;
-  height: 465px;
-  left: 24px;
-  top: 24px; 
-}
+  .seeproject {
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+  }
 
-.paragraph2 {
-  width: 496px;
-  height: 70px;
-  font-weight: 400;
-  font-size: 16px;
-  line-height: 24px;
-}
+  .card2 {
+    display: flex;
+    flex-direction: row;
+    width: 1156px;
+  }
 
-.aboutme {
-  display: flex;
-  flex-direction: row;
-  align-items: flex-start;
-  justify-content: space-between;
-  width: 85%;
-  height: auto;
-  /* padding: 10% 10%; */
-  left: 1%;
-  /* top: 3700px; */
-}
+  .avator4 {
+    width: 544px;
+    height: 448px;
+    left: 590px;
+    top: 24px;
+    order: 2;
+  }
 
-.top {
-  /* margin: auto; */
-  width: 50%;
-}
+  .container2 {
+    order: 1;
+    padding: 24px;
+    gap: 24px;
+    width: 544px;
+    height: 465px;
+    left: 24px;
+    top: 24px;
+  }
 
-.heading {
-  width: 50%;
-  height: auto;
-  font-weight: 700;
-  font-size: 40px;
-  line-height: 52px;
-}
+  .aboutme {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: space-between;
+    left: 0;
+    width: 1440px;
+    height: 924px;
+    background-image: url("./sources/DesktopBackground/about\ myself\ -\ background-\ shape-\ desktop.png");
+  }
```

Comment:
- [ ] I would kindly suggest you stick to the Figma design as much as possible as this will show your consistency and ability to replicate any given project. currently, the `about` section of your project is not aligning well with the languages as seen in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=QGQEFWn9azsQGUAb-0)

- Kindly checkout my outlines below and fix them accordingly.

Your design 👇 

![joy abt](https://user-images.githubusercontent.com/78634841/208218289-fbd778db-e68e-4de5-8632-bf46ea4285e2.png)

Figma Design 👇 

![abut real](https://user-images.githubusercontent.com/78634841/208218327-6aa39522-f615-41f4-945e-8fd1c7f7946d.png)